.env.example

```diff
@@ -34,6 +34,7 @@ LNBITS_SITE_TAGLINE="free and open-source lightning wallet"
 LNBITS_SITE_DESCRIPTION="Some description about your service, will display if title is not 'LNbits'"
 # Choose from mint, flamingo, freedom, salvador, autumn, monochrome, classic
 LNBITS_THEME_OPTIONS="classic, bitcoin, freedom, mint, autumn, monochrome, salvador"
+LNBITS_CUSTOM_LOGO="https://lnbits.com/assets/images/logo/logo.svg"
```

Comment:
This should be commented out otherwise all users will pull this svg from lnbits.com
```suggestion
# LNBITS_CUSTOM_LOGO="https://lnbits.com/assets/images/logo/logo.svg"
```